android/src/main/java/io/sentry/react/RNSentryModule.java

```diff
@@ -198,25 +198,24 @@ public void fetchNativeRelease(Promise promise) {
     @ReactMethod
     public void fetchNativeAppStart(Promise promise) {
         final AppStartState appStartInstance = AppStartState.getInstance();
-        final Date appStartTime = appStartInstance.getAppStartTime();
+        final Long appStartInterval = appStartInstance.getAppStartInterval();
 
-        if (appStartTime == null) {
+        if (appStartInterval == null && didFetchAppStart) {
```

Comment:
no need to be sending the app start `map` back to JS every time if already sent, this should be done on iOS too